shared/InputField/input_field.tsx

```diff
@@ -31,7 +33,12 @@ const AppInputField = ({
         {!isEditable ? (
           <FaPlus onClick={() => onSubmit()} />
         ) : (
-          <FaCheck onClick={() => onSubmit()} />
+          <FaCheck
+            onClick={() => {
+              onSubmit();
+              handleEditClick();
+            }}
+          />
```

Comment:
Move to callback

Reply:
To custom hook? or just create a new function that includes both of them